script/swagger

```diff
@@ -14,6 +14,10 @@ fi
 
 (
   cd swagger-codegen
+  if [[ "${#SWAGGER_CODGEN_REVISION}" -lt 40 ]]; then
```

Comment:
This is how you get the [length of a string in bash](http://stackoverflow.com/a/17368090/454265).